config/templates/cluster.yaml

```diff
@@ -121,6 +130,18 @@ worker:
 # (Set to an odd number >= 3 for HA control plane)
 # etcdCount: 1
 
+# etcd:
+#   privateSubnets:
+#     - availabilityZone: us-west-1a
+#       instanceCIDR: "10.0.2.0/24"
+#     - availabilityZone: us-west-1b
+#       instanceCIDR: "10.0.3.0/24"
+#       # natGateway:
+#       #   # Pre-allocated NAT Gateway. Used with private subnets.
+#       #   id: "ngw-abcdef12"
+#       #   # Pre-allocated EIP for NAT Gateways. Used with private subnets.
+#       #   eipAllocationIDs: "eipalloc-abcdef12"
```

Comment:
nit: Intended `eipAllocationId` rather than `eipAllocationIDs`? For historical reason(😝), it is `Id` rather than `ID`. You'd assign only one EIP to a NAT gateway hence singular `Id` instead of plural `IDs`.